todoman/cli.py

```diff
@@ -466,6 +468,8 @@ def move(ctx, list, ids):
 @cli.command()
 @pass_ctx
 @click.argument('lists', nargs=-1, callback=_validate_lists_param)
+@click.option('--description',
```

Comment:
I really don't understand which tests are missing.
And if I remove the flag then also it works correctly then why that implementation was not wrong.